src/pages/auth/login/Login.jsx

```diff
@@ -0,0 +1,104 @@
+import "./Login.css";
+import { Link } from "react-router-dom";
+import { MdDoubleArrow } from "react-icons/md";
+import { BsPersonBadge } from "react-icons/bs";
+import { useAuth } from "../../../context/auth-context";
+
+export function Login() {
+  const { authState, loginHandler } = useAuth();
+  const { error } = authState;
+
+  const submitHandler = (e) => {
+    e.preventDefault();
+
+    const form = e.target;
+    const [email, password] = e.target.elements;
+
+    if (!form.checkValidity()) form.classList.add("validate");
+    else {
+      loginHandler(email.value, password.value);
+    }
+  };
+
+  const guestLoginHandler = () => {
+    // Implementing guest login
+    loginHandler("adarshbalika@gmail.com", "adarshBalika123");
+  };
```

Comment:
Really good implementation of the guest signin functionality in one line!!Good work @tusharkandpal

Reply:
thanks @dev-enforced !